website/versioned_docs/version-0.20/concerto-cli.md

```diff
@@ -85,12 +85,12 @@ Options:
 ```
 
 At the moment, the available target formats are as follows:
-- Go Lang: `concerto compile --ctoFiles modelfile.cto --format Go`
-- Plant UML: `concerto compile --ctoFiles modelfile.cto --format PlantUML`
-- Typescript: `concerto compile --ctoFiles modelfile.cto --format Typescript`
-- Java: `concerto compile --ctoFiles modelfile.cto --format Java`
-- JSONSchema: `concerto compile --ctoFiles modelfile.cto --format JSONSchema`
-- XMLSchema: `concerto compile --ctoFiles modelfile.cto --format XMLSchema`
+- Go Lang: `concerto compile --ctoFiles modelfile.cto --target Go`
+- Plant UML: `concerto compile --ctoFiles modelfile.cto --target PlantUML`
+- Typescript: `concerto compile --ctoFiles modelfile.cto --target Typescript`
+- Java: `concerto compile --ctoFiles modelfile.cto --target Java`
+- JSONSchema: `concerto compile --ctoFiles modelfile.cto --target JSONSchema`
+- XMLSchema: `concerto compile --ctoFiles modelfile.cto --target XMLSchema`
```

Comment:
I believe this doesn't need to be changed, and rather will be auto-generated by the changes made to `docs/concerto-cli.md` so these changes can be removed. @jeromesimeon confirm this?

Reply:
yes and no (the versioning should be done through `npm run version` though but I can clean that up after it's been merged in.